config/flags_network.go

```diff
@@ -34,11 +34,18 @@ var (
 		Value: string(Mainnet),
 	}
 	// FlagAPIAddress Mysterium API URL
+	// Deprecated: use FlagDiscoveryAddress
 	FlagAPIAddress = cli.StringFlag{
 		Name:  metadata.FlagNames.MysteriumAPIAddress,
 		Usage: "URL of Mysterium API",
```

Comment:
Add `Deprecated: use FlagDiscoveryAddress` into this usage too, so users will see it once it's compiled.